app/models/value_definition.rb

```diff
@@ -4,10 +4,20 @@
 class ValueDefinition < ApplicationRecord
   belongs_to :benchmark, class_name: 'Xccdf::Benchmark'
 
-  POSSIBLE_VALUE_TYPES = %w[string number bool].freeze
+  POSSIBLE_VALUE_TYPES = %w[string number boolean].freeze
 
   validates :benchmark_id, presence: true
   validates :ref_id, uniqueness: { scope: %i[benchmark_id] }, presence: true
   validates :description, presence: true
   validates :value_type, presence: true, inclusion: { in: POSSIBLE_VALUE_TYPES }
+
+  def self.from_openscap_parser(op_vd, existing: nil, benchmark_id: nil)
+    value_definition = existing || new(ref_id: op_vd.id, benchmark_id: benchmark_id)
+
+    value_definition.assign_attributes(title: op_vd.title, description: op_vd.description,
+                                       value_type: op_vd.type, default_value: op_vd.value,
+                                       lower_bound: op_vd.lower_bound.to_f, upper_bound: op_vd.upper_bound.to_f)
```

Comment:
Do we really need these last two?

Reply:
No, not really. I was going to ask you about this. I think we can leave lower_bound and upper_bound off until openscap decides they want to start using them(which they said they aren't planning on doing)